apps/condo/domains/ticket/schema/TicketExportTask.test.js

```diff
@@ -136,16 +200,15 @@ describe('TicketExportTask', () => {
         it('can be updated by user with specifying "cancelled" value for "status" field', async () => {
             const adminClient = await makeLoggedInAdminClient()
             const userClient = await makeClientWithNewRegisteredAndLoggedInUser()
-            const [obj] = await createTestTicketExportTask(adminClient, adminClient.user, {
-                status: PROCESSING,
-            })
+            const [obj] = await createTestTicketExportTask(adminClient, adminClient.user)
             const [objUpdated] = await updateTestTicketExportTask(userClient, obj.id, { status: CANCELLED })
+            // We need to wait until worker that is launched in `afterChange` hook will be finished
+            await sleep(2000)
```

Comment:
?

Reply:
Worker sets status to `completed`, we need to avoid concurrency.